glean_parser/schemas/pings.1-0-0.schema.yaml

```diff
@@ -0,0 +1,60 @@
+$schema: http://json-schema.org/draft-07/schema#
+title: Pings
+description: |
+  Schema for the pings.yaml files for Mozilla's glean telemetry SDK.
+
+  The top-level of the `pings.yaml` file has a key defining the name of each
+  ping. The values contain metadata about that ping. Ping names must be
+  snake_case, though they may also have dots `.` to define subcategories.
+
+$id: moz://mozilla.org/schemas/glean/pings/1-0-0
+
+definitions:
+  dotted_snake_case:
+    type: string
+    pattern: "^[a-z_][a-z0-9_]{0,29}(\\.[a-z_][a-z0-9_]{0,29})*$"
+    maxLength: 40
+
+type: object
+
+propertyNames:
+  anyOf:
+    - $ref: "#/definitions/dotted_snake_case"
+    - enum: ['$schema']
+
+properties:
+  $schema:
+    type: string
+    format: url
+
+additionalProperties:
+  type: object
+  properties:
+    description:
+      title: Description
+      description: |
+        **Required.**
+
+        A textual description of the purpose of this ping and what it contains.
+      type: string
+
+    include_client_info:
+      title: Include client info
+      description: |
+        **Required.**
+
+        When `true`, include the `client_info` block in the ping.
+      type: boolean
+
+    disabled:
+      title: Disabled
+      description: |
+        If `true`, the ping is disabled, and will not be sent.
+      type: boolean
+      default: false
```

Comment:
As Travis mentioned in the bug, maybe we could/should add data review-related metadata here as well.  `data_review`, `notification_emails`, `bugs`...?

Reply:
Yes, but this is probably material for a proposal document. I suggest we do the following: keep the properties for just the `client_id` and the description, file a follow-up bug for defining which metadata should go in here, have a proposal and let it fly by the rest of the glean team.

Reply:
+1 for this, and the proposal